talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+def extract_MFCC(file, outPath):
+	# CPU: extract mfcc
+	sr, audio = wavfile.read(file)
+	mfcc = python_speech_features.mfcc(audio,sr) # (N_frames, 13)   [1s = 100 frames]
+	featuresPath = os.path.join(outPath, file.split('/')[-1].replace('.wav', '.npy'))
+	numpy.save(featuresPath, mfcc)
+
+def evaluate_network(s, files):
+	# GPU: active speaker detection by pretrained TalkNet
+	allScores = []
+	# durationSet = {1,2,4,6} # To make the result more reliable
+	durationSet = {1,1,1,2,2,2,3,3,4,5,6} # Use this line can get more reliable result
```

Ruff (0.13.3)

Sets should not contain duplicate item `1`

Remove duplicate item

(B033)